src/ir_Fujitsu.cpp

```diff
@@ -0,0 +1,220 @@
+// Copyright 2017 Jonny Graham
+#include "ir_Fujitsu.h"
+#include <algorithm>
+#include "IRrecv.h"
+#include "IRsend.h"
+#include "IRtimer.h"
+
+// Fujitsu A/C support added by Jonny Graham
+
+
+// Fujitsu A/C
+// Ref:
+// These values are based on averages of measurements
+#define FUJITSU_AC_HDR_MARK    3224U
+#define FUJITSU_AC_HDR_SPACE   1574U
+#define FUJITSU_AC_BIT_MARK    448U
+#define FUJITSU_AC_ONE_SPACE   1182U
+#define FUJITSU_AC_ZERO_SPACE  367U
+#define FUJITSU_AC_TRL_MARK    448U
+#define FUJITSU_AC_TRL_SPACE   8100U
+
+#if SEND_FUJITSU_AC
+// Send a Fujitsu A/C message.
+//
+// Args:
+//   data: An array of bytes containing the IR command.
+//   nbytes: Nr. of bytes of data in the array. (>=FUJITSU_AC_STATE_LENGTH)
```

Comment:
I just realised this comment is incorrect, as you can send the smaller size.
Probably should say it's typically FUJITSU_AC_STATE_LENGTH_SHORT or FUJITSU_AC_STATE_LENGTH